docs/README.ios.md

```diff
@@ -0,0 +1,227 @@
+codecs that support a multitude of music and video formats.
+
+The minimum version of iOS you need to run(!) Kodi is 5.1.1 atm.
+
+- On Mavericks (OSX 10.9.x) we recommend using Xcode 6.1.
+- On Yosemite (OSX 10.10.x) we recommend using Xcode 6.4.
+- On El Capitan (OSX 10.11.x) we recommend using Xcode 7.x or Xcode 8.x.
+- On Sierra (macOS 10.12.x) we recomment using Xcode 8.x.
+
+## 2 Getting the source code
+
+    cd $HOME
+    git clone git://github.com/xbmc/xbmc.git Kodi
+
+## 3 Install build dependencies
+
+### 3.1 Install Xcode
+
+Install the Xcode version recommended for your macOS version. You can download
+it either from the macOS AppStore (Xcode) or from the Apple Developer Homepage.
+
+As far as we know the compilation for iOS should work with the following
+constellations of Xcode and macOS versions (to be updated once we know more):
+
+1. XCode 6.0.1 against iOS SDK 8.0 on 10.9 (Mavericks)
+2. XCode 6.1.0 against iOS SDK 8.1 on 10.10 (Yosemite)
+3. XCode 6.3.0 against iOS SDK 8.3 on 10.10 (Yosemite)
+4. Xcode 6.4.0 against iOS SDK 8.4 on 10.10 (Yosemite)
+5. Xcode 7.x against iOS SDK 9.x on 10.10 (Yosemite)
+6. Xcode 7.x against iOS SDK 9.x on 10.11 (El Capitan)
+7. Xcode 7.x against iOS SDK 9.x on 10.12 (Sierra)
+
+The preferred iOS SDK Version is 8.1.
+
+### 3.2 Install Kodi build depends
+
+Kodi requires a set of build dependencies to be built and installed before you
+will be able to build the Kodi main binary. These often just called *depends*
+are installed using the commands described below (with the latest iOS SDK found
+on your system).
+
+In order to speedup compilation it is recommended to use `make -j$(getconf
+_NPROCESSORS_ONLN)` instead of `make` to compile on all available processor
+cores. The build machine can also be configured to do this automatically by
+adding `export MAKEFLAGS="-j(getconf _NPROCESSORS_ONLN)"` to your shell config
+(e.g. `~/.bashrc`).
+
+#### 3.2.a Compiling as 32 bit armv7 libraries (recommended for most users)
+
+    cd $HOME/Kodi
+    cd tools/depends
+    ./bootstrap
+    ./configure --host=arm-apple-darwin
+    make
+
+#### 3.2.b Compiling as 64 bit arm64 libraries
+
+    cd $HOME/Kodi
+    cd tools/depends
+    ./bootstrap
+    ./configure --host=arm-apple-darwin --with-cpu=arm64
+    make
+
+#### 3.3.c Advanced topics
+
+The dependencies are built into `tools/depends` and installed into
+`/Users/Shared/xbmc-depends`.
+
+**ADVANCED developers only**: If you want to specify an iOS SDK version (if
+multiple versions are installed) - then append it to the configure line
+above. The example below would use the iOS SDK 8.0:
+
+    ./configure --host=arm-apple-darwin --with-sdk=8.0
+
+Ensure that you also adapt the Xcode project to use this SDK version.
+
+### 3.3 Compile Kodi binary addons
+
+Kodi maintains a set of binary addons (PVR clients, Visualizations, Audio DSP
+plugins and more). They can be built as shown below:
+
+    make -C target/binary-addons
```

Comment:
this is out of context now - please ensure that people are in tools/depends before trying to issue this command.